src/components/button.component.js

```diff
@@ -188,7 +191,12 @@ export class Button extends Component {
         }}
         disabledStyle={disabledStyle}
         icon={
-          icon && { ...defaultIconStyle, ...sizes[size].iconStyle, ...icon }
+          icon && {
+            ...defaultIconStyle,
```

Comment:
Extended this since didn't seem like icons were being styled in the component within @machour